functional_tests/lightning/train-gpu-ddp.py

```diff
@@ -28,15 +35,15 @@ def main():
 
```

Comment:
Can you please add `name=__file__` to the logger instantiation above? (so that it is easier to navigate test results in the UI)?

Reply:
fixed. good point